### ISSUE TITLE
Error upgrading to 7.0.0 via choco

### ISSUE THREAD

**Issue by antivirtel** [rcmaehl/MSEdgeRedirect#155]
Hi, 

I'm on 0.6.5.0, trying to upgrade to 0.7.0.0, but choco seems to fail. I'm on Win10, latest patches.
```choco
You have msedgeredirect v0.6.5.0 installed. Version 0.7.0.0 is available based on your source(s).
Progress: Downloading msedgeredirect 0.7.0.0... 100%

msedgeredirect v0.7.0.0 [Approved]
msedgeredirect package files upgrade completed. Performing other installation steps.
File appears to be downloaded already. Verifying with package checksum to determine if it needs to be redownloaded.
Hashes match.
Hashes match.
Installing msedgeredirect...
ERROR: Running ["C:\Users\user\AppData\Local\Temp\chocolatey\msedgeredirect\0.7.0.0\MSEdgeRedirect.exe" /wingetinstall ] was not successful. Exit code was '160'. See log for possible error messages.
The upgrade of msedgeredirect was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\msedgeredirect\tools\chocolateyinstall.ps1'.
 See log for details.

...

Chocolatey upgraded 0/50 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - msedgeredirect (exited 160) - Error while running 'C:\ProgramData\chocolatey\lib\msedgeredirect\tools\chocolateyinstall.ps1'.
 See log for details.
```

Is there anything on my side I can do, or do you need to fix it?

Thanks!

**Comment by rcmaehl** [comment]
Odd. 160 shouldn't be triggered for `/wingetinstall`. Let me review some stuff.

**Comment by rcmaehl** [comment]
Hi @antivirtel. This should be fixed for 0.7.0.1. Let me know if it isn't.

**Comment by antivirtel** [comment]
Thanks, works now. Hope it's accepted soon, but I upgraded with the `--version` flag.
